### ISSUE TITLE
Certify / Regulatory approvals for PPE visors

### ISSUE THREAD

**Issue by ajlennon** [DoESLiverpool/covid19#24]
Speaking with Dave Kershaw who is going to work with us to try to certify or at least ensure maximal regulatory compliance for PPE provision

Placeholder for those conversations

**Comment by amcewen** [comment]
The [EU has made their standards specs for medical supplies free to download](https://ec.europa.eu/commission/presscorner/detail/en/ip_20_502)

**Comment by ajlennon** [comment]
@AlyssaAbalassi provided this which seems relevant in this situation

[https://www.gov.uk/guidance/exceptional-use-of-non-ce-marked-medical-devices](https://www.gov.uk/guidance/exceptional-use-of-non-ce-marked-medical-devices)

**Comment by amcewen** [comment]
The [BSI standards are here](https://www.bsigroup.com/en-GB/topics/novel-coronavirus-covid-19/medical-devices-ppe/) (I'm chatting to a mate who's at the Office of Product Safety and Standards).  And there's also https://www.gov.uk/government/news/regulations-temporarily-suspended-to-fast-track-supplies-of-ppe-to-nhs-staff-and-protect-companies-hit-by-covid-19

**Comment by ajlennon** [comment]
Have looped in @DaveKershaw on this.

**Comment by drakard** [comment]
Paul Bullock is now focusing on this area.

**Comment by amcewen** [comment]
My mate at the Office of Product Safety and Standards says:
> You should e-mail opss.enquiries@beis.gov.uk about your 3D printed visor. They'll be able to advise about whether they can grant an "easement" from some of the regulations, allowing you to distribute.

**Comment by DaveKershaw** [comment]
This is correct however you have to apply for the easement. Strictly speaking the product should be CE marked before it can be supplied. For low risk products this can be by self certification. However from the classification process in the PPE regulation the classification of the product is Class III and these need notified body approval befor the manufacturer can place the CE mark on the product. The interesting thing is that as far as I can find out you still have to comply with the regulation other than the final CE marking stage where notified body approval is required before the product can be CE marked. 
To cover the collective backside it would be advisable to comply with the risk management, manufacturing, product technical file requirements so that it can be demonstrated that the product was not just thrown together (which it isn’t but its always useful to have paper to defend ones actions). The greater the rigour in the design development & manufacture the easier the case is to make for the easement.
Cheers
Dave

**Comment by plastictactics** [comment]
Has anyone started to put together a tech file? I have some experience in self certified CE marking so could get the ball rolling - but would like someone closer to the project to give me the nod rather than risk duplication.
^arthur

**Comment by ajlennon** [comment]
We've got a chap David Kershaw who has started looking at this - would be great if you could liaise with him and start putting the tech file together @plastictactics

**Comment by plastictactics** [comment]
Thanks @ajlennon - could you make an intro between myself and David by email? arthur@arthurrowland.com

edit: Just noticed that @DaveKershaw is in this thread - might be better for us to keep things on this github rather than go offsite. Dave, please let me know where you're up to and how I might help.

**Comment by ajlennon** [comment]
Hi @plastictactics - OK so background is Max Zadow and I put a grant bid into Helpful Engineering a week last Friday and won their competition which was nice!

see: [https://docs.google.com/document/d/1edP_js_qHgL57S_6JvaYwVUFfh2x-DbDntAcT4gygls/edit?usp=sharing](https://docs.google.com/document/d/1edP_js_qHgL57S_6JvaYwVUFfh2x-DbDntAcT4gygls/edit?usp=sharing)

They invited us to put another bid into a sister organisation of theirs JOGL which I did Sunday night. Project is here (needs reworking) [https://app.jogl.io/project/186](https://app.jogl.io/project/186)

The initial bid was well received. We did have some criticism around regulatory approvals and so forth. @DaveKershaw is very up on all this and Max asked him to add some content which he kindly provided but I haven't had time to integrate into the JOGL project. I will email that over with the introduction. If you were able to take that on and add it to the JOGL project that would be awesome

**Comment by plastictactics** [comment]
OK thanks for the context - will take a look at the detail tomorrow morning and see if I can help. Are these applications for fundraising only, or are there other benefits?

**Comment by ajlennon** [comment]
Fundraising and awareness raising

**Comment by ajlennon** [comment]
*but* that said @DaveKershaw 's input has a much more important and wider value in terms of overall design development / certification and what you were mentioning with the technical file. The JOGL additions were just a starting point to try to get us aligned on the right track if that makes any sense (it's late and it's been a long day ;) )

**Comment by drakard** [comment]
I believe 3D crowd UK have been doing work in this area, as have 3P Innovations.

**Comment by ajlennon** [comment]
I think 3DCrowd have just signed something with BSI on 3DPrinting and injection moulding

**Comment by drakard** [comment]
Thanks Alex. I just sent an email to 3DCrowd asking for a contact on the approvals work.

**Comment by drakard** [comment]
This agrees with what we've heard before, that wide visors are desirable. https://www.bbc.co.uk/news/technology-52619138

The article mentions A4, the size we use, may not be wide enough. To the best of my knowledge the A4 size we use is already one of the wider designs. It appears, although it was not stated, that the actual length of the visor material might not be the only factor, but the design of how it wraps round the face could have an effect.
